frontend/controller/router.js

```diff
@@ -87,8 +87,8 @@ const router = new Router({
     },
     {
       path: '/design-system',
-      component: DesignSystem,
-      name: DesignSystem.name,
+      component: () => DesignSystem,
+      name: 'DesignSystem',
```

Comment:
Interesting idea. I'm on the fence personally in terms of whether to use `DesignSystem.name` or `'DesignSystem'`. The first one is safer against renames, and facilitates project-wide renames in the IDE. But the second one is less "magical", and allows you to move the arrow function back up into the `const`s above, which I think is technically more "correct". Just some thoughts, not an official request for changes. I think a case could be made for both sides on this.